### DISCUSSION TITLE
[Feature] support the video/avi file type too?

### DISCUSSION THREAD

**Discussion by mkjustuk** [immich-app/immich#2964] [Feature Request]
### The feature

I saw a previous issue that looked like it had covered this, but I think it only covered a different type of .avi files.
The video type video/avi is not recognised and would be great if it could be added.

I get this in the logs on server:

[Nest] 1  - 06/26/2023, 12:23:36 PM   ERROR [AssetUploadConfig] Unsupported file type .AVI file MIME type video/avi

Happy to send a sample vid etc if needed.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by uhthomas** [comment]
This is a regression I believe, apologies. We were speaking about this last night and need to add a few missing mime types. AVI has four!

https://en.wikipedia.org/wiki/Audio_Video_Interleave

We did remove some incorrect mime types though, like `video/mov` is not valid.

**Reply by mkjustuk** [reply]
Four, wow.  No apologies needed at all, these were the first AVI types I've had. I'm loving that I can add my MTS videos natively that is a massive help.

**Comment by NicholasFlamy** [comment]
Should this be closed? I'm able to upload an AVI file to the server.

(Can't play it in my browser but that's a browser issue. Though I'm confused as to why it's stuck loading rather than playing the transcoded video. Seemingly not detecting unsupported format in the browser and loading the transcoded file.)

**Reply by zdimension** [reply]
> (Can't play it in my browser but that's a browser issue. Though I'm confused as to why it's stuck loading rather than playing the transcoded video. Seemingly not detecting unsupported format in the browser and loading the transcoded file.)

Having the same issue here, AVI file not playing in browser but working in the Android app.
